argilla/docs/tutorials/image_classification.ipynb

```diff
@@ -0,0 +1,804 @@
+{
```

Comment:
<p>Do you think it would be worth adding a feature to load the public url of the image? We could limit this hf datasets in <code>log</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/argilla-io/argilla/pull/5413/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
@burtenshaw I think it could be worth it but not every Hugging Face dataset has a public URL and not every image can be loaded as a working base64 encoded string. I think such workflows can be added specifically to the dataset config documentation or as an advanced guide.